### DISCUSSION TITLE
Statistics under construction

### DISCUSSION THREAD

**Discussion by tphakala** [tphakala/birdnet-go#244] [General]
Just a heads up that I have started work on adding statistics, at the moment plan is to render statistics with Apache ECharts using  [go-echarts library
](https://github.com/go-echarts/go-echarts)

I managed to cobble together a following responsive chart with help of Claude.AI

![image](https://github.com/tphakala/birdnet-go/assets/7030001/aea001dc-f813-47ed-bbee-2560311694f5)

Once I manage to make anything useful out of this I will push it into github.

**Comment by isZumpo** [comment]
Looks like a great start to me! Wonder if grafana could be used for some rapid prototyping of what graphs are useful or not for birds. Not sure if the current prometheus metrics exposes enough metrics to be useful for that purpose.

**Comment by have-no-clue-what-im-doing** [comment]
Can't wait!!!

**Comment by 0n3man** [comment]
The graph looks good.  A multi day line graph with different color line for each bird might be another option.
